src/index.js

```
@@ -1 +1,16 @@
 'use strict';
+require('dotenv').config({ path: '../.env' });
+require('./models/user');
+
+const { createServer } = require('./createServer');
+const { sequelize } = require('./utils/db');
+
+(async () => {
+  try {
+    await sequelize.authenticate();
+
+    await sequelize.sync({ alter: true });
+
+    createServer().listen(5700, () => {});
+  } catch (error) {}
```

Critical: current catch swallows errors (`catch (error) {}`) which hides startup failures (DB connection, sync, etc.). Log the error and exit or rethrow so issues are visible (e.g., `console.error(error); process.exit(1);`).